### ISSUE TITLE
[e2e-cadvisor-rhel-7] e2e flake: Permission denied

### ISSUE THREAD

**Issue by timstclair** [google/cadvisor#1228]
https://console.cloud.google.com/storage/browser/kubernetes-jenkins/logs/cadvisor-gce-e2e-ci/594

```
E0409 11:07:59.234503   13848 framework.go:335] About to run - [ssh -i /home/jenkins/.ssh/google_compute_engine -o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o CheckHostIP=no -o StrictHostKeyChecking=no e2e-cadvisor-rhel-7 -- sh -c " sudo docker rm -f 27e1929b4a8d45726ae65f705f846d509167640c5592dc055ae35fac9dbb0a0c "]
--- FAIL: TestDockerContainerMemoryStats (1.15s)
        framework.go:338: Failed to run "sudo" [docker rm -f 27e1929b4a8d45726ae65f705f846d509167640c5592dc055ae35fac9dbb0a0c] in "e2e-cadvisor-rhel-7" with error: "exit status 255". Stdout: "", Stderr: Warning: Permanently added 'e2e-cadvisor-rhel-7' (ECDSA) to the list of known hosts.
                Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
```

https://console.cloud.google.com/storage/browser/kubernetes-jenkins/logs/cadvisor-gce-e2e-ci/656

```
E0410 18:08:07.406428    8306 framework.go:335] About to run - [ssh -i /home/jenkins/.ssh/google_compute_engine -o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o CheckHostIP=no -o StrictHostKeyChecking=no e2e-cadvisor-rhel-7 -- sh -c " sudo docker run -d busybox ping www.google.com "]
--- FAIL: TestDockerContainerCpuStats (0.17s)
        framework.go:338: Failed to run "sudo" [docker run -d busybox ping www.google.com] in "e2e-cadvisor-rhel-7" with error: "exit status 255". Stdout: "", Stderr: Warning: Permanently added 'e2e-cadvisor-rhel-7' (ECDSA) to the list of known hosts.
                Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
```

https://console.cloud.google.com/storage/browser/kubernetes-jenkins/logs/cadvisor-gce-e2e-ci/1003

```
E0417 20:09:14.644640   29671 framework.go:335] About to run - [ssh -i /home/jenkins/.ssh/google_compute_engine -o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o CheckHostIP=no -o StrictHostKeyChecking=no e2e-cadvisor-rhel-7 -- sh -c " sudo docker rm -f c85813caefbe854d03e86ccd1bdae253770af70402429e3539a00f00b3bed77a "]
--- FAIL: TestDockerContainerCpuStats (1.12s)
        framework.go:338: Failed to run "sudo" [docker rm -f c85813caefbe854d03e86ccd1bdae253770af70402429e3539a00f00b3bed77a] in "e2e-cadvisor-rhel-7" with error: "exit status 255". Stdout: "", Stderr: Warning: Permanently added 'e2e-cadvisor-rhel-7' (ECDSA) to the list of known hosts.
                Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
```

/cc @sjenning @pwittrock

**Comment by timstclair** [comment]
Another:
https://console.cloud.google.com/storage/browser/kubernetes-jenkins/logs/cadvisor-gce-e2e-ci/1045
```
E0418 17:07:24.267037   11652 framework.go:335] About to run - [ssh -i /home/jenkins/.ssh/google_compute_engine -o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o CheckHostIP=no -o StrictHostKeyChecking=no e2e-cadvisor-rhel-7 -- sh -c " sudo docker rm -f 9749032b01832053d81b14607e2efc527b653869cb3af8038eb47e51481a3a7a "]
--- FAIL: TestGetAllDockerContainers (8.42s)
        framework.go:338: Failed to run "sudo" [docker run -d kubernetes/pause] in "e2e-cadvisor-rhel-7" with error: "exit status 255". Stdout: "", Stderr: Warning: Permanently added 'e2e-cadvisor-rhel-7' (ECDSA) to the list of known hosts.
                Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
```

**Comment by timstclair** [comment]
https://console.cloud.google.com/storage/browser/kubernetes-jenkins/logs/cadvisor-gce-e2e-ci/1831

**Comment by timstclair** [comment]
This recently started happening on `container-vm` as well:
https://console.cloud.google.com/storage/browser/kubernetes-jenkins/logs/cadvisor-gce-e2e-ci/6694
```
framework.go:338: Failed to run "sudo" [docker rm -f e651803f0c740863434cbb4b6cc48048959f07dcc14c3fc469e0b91b1b78cc10] in "e2e-cadvisor-container-vm-v20160127" with error: "exit status 255". Stdout: "", Stderr: Warning: Permanently added 'e2e-cadvisor-container-vm-v20160127' (ECDSA) to the list of known hosts.
                Permission denied (publickey).
```

https://console.cloud.google.com/storage/browser/kubernetes-jenkins/logs/cadvisor-gce-e2e-ci/6768
```
framework.go:338: Failed to run "sudo" [docker run -d kubernetes/pause] in "e2e-cadvisor-container-vm-v20151215" with error: "exit status 255". Stdout: "", Stderr: Warning: Permanently added 'e2e-cadvisor-container-vm-v20151215' (ECDSA) to the list of known hosts.
                Permission denied (publickey).
```

**Comment by dashpole** [comment]
I think this was "resolved" by #1327.  Either way, this is almost a year old...
